src/ProjectNeon/Assets/Scripts/HPBarController.cs

```diff
@@ -0,0 +1,75 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+public class HPBarController : MonoBehaviour
+{
+    #region Parametrs
+    [SerializeField] Image barImage;
+    [SerializeField] Text barTextValue;
+
+    int maxHP = 100;
+
+    public int currentHP { get; private set; }
```

Comment:
All public properties should be pascal-cased, as per C# conventions. `CurrentHP`